### DISCUSSION TITLE
Captions within tables and List of Tables

### DISCUSSION THREAD

**Discussion by ghost** [plutext/docx4j#498] [General]
Hi,

Does the library support List of Tables, in case a table has a caption?

From what I've see in the documentation https://docx4java.org/docx4j/Docx4j_GettingStarted.pdf, page 36 it doesn't

Thanks

**Comment by plutext** [comment]
If you unzip a docx and inspect it, you'll see this is done with a SEQ in the caption, together with a TOC with \c switch referencing the sequence.  

Higher level support, as of 11.5.4; see https://github.com/plutext/docx4j/discussions/486

(There has never been anything stopping you from working with this content in a low-level way)
